src/utils/sanity.fetch.only.server.ts

```diff
@@ -24,6 +24,7 @@ export async function sanityQuery<T = any>(query: string): Promise<T> {
       headers: {
         Authorization: `Bearer ${process.env.SANITY_API_TOKEN}`,
       },
+      next: {revalidate: 3600},
```

Comment:
this is an hour, which is a long time if you are expecting an edit to show up

Reply:
I'm not sure what time to set it as. Because in my mind, edits should show up immediately